### PR TITLE
Discard the 'buffering' argument when instantiating an HTTPResponse on python 2.6.

### PR DESCRIPTION
**Pull request by cazino**
Python 2.6 httlib.HTTPResponse  does not accept a 'buffering' arg.

### CODE REVIEW

File: test/unit/test_http_utils.py [hunk 1]
```diff
@@ -116,3 +118,21 @@ def server(client, _):
                 self.assertEqual(''.join(rest), data[off:])
         finally:
             server_thread.kill()
+
+
+class TestSomewhatBufferedHTTPConnection(unittest.TestCase):
+
+    def test_discard_buffering_arg_on_py26(self):
+        with mock.patch('httplib.HTTPResponse.__init__') \
```
Comment (JordanP) [inline test/unit/test_http_utils.py]: You should use the decorator form of `patch`
```
@mock.patch('httplib.HTTPResponse.__init__')
def test_discard_buffering_arg_on_py26(self, mock_http_response_init)
```

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/swift-tox/179/<h2>Build result: FAILURE</span></h2>[...truncated 115 lines...]Triggering swift-tox » py27-swifthead,ubuntu14-largeTriggering swift-tox » py27-swift2.2.0,centos7-largeTriggering swift-tox » py26-swifthead,centos6-largeTriggering swift-tox » pep8,ubuntu14-largeTriggering swift-tox » py27-swift2.2.0,ubuntu14-largeTriggering swift-tox » py26-swift1.13.1,centos6-largeTriggering swift-tox » py27-swifthead,ubuntu12-largeConfiguration swift-tox » py27-swift1.13.1,centos7-large is still in the queue: All nodes of label ‘centos7-large’ are offlineConfiguration swift-tox » py27-swift1.13.1,centos7-large is still in the queue: Waiting for next available executor on centos7-largeConfiguration swift-tox » py27-swift1.13.1,centos7-large is still in the queue: All nodes of label ‘centos7-large’ are offlineswift-tox » py27-swift1.13.1,centos7-large completed with result SUCCESSswift-tox » py27-swifthead,centos7-large completed with result SUCCESSswift-tox » py27-swift1.13.1,ubuntu12-large completed with result FAILUREswift-tox » py27-swift1.13.1,ubuntu14-large completed with result SUCCESSswift-tox » py27-swift2.2.0,ubuntu12-large completed with result FAILUREswift-tox » py26-swift2.2.0,centos6-large completed with result SUCCESSswift-tox » py27-swifthead,ubuntu14-large completed with result SUCCESSswift-tox » py27-swift2.2.0,centos7-large completed with result SUCCESSswift-tox » py26-swifthead,centos6-large completed with result SUCCESSswift-tox » pep8,ubuntu14-large completed with result SUCCESSswift-tox » py27-swift2.2.0,ubuntu14-large completed with result SUCCESSswift-tox » py26-swift1.13.1,centos6-large completed with result SUCCESSswift-tox » py27-swifthead,ubuntu12-large completed with result FAILURESetting status of 325353f63c5120cf351f32d79ca59a2429d55f6e to FAILURE with url https://37.187.159.67:5443/job/swift-tox/179/ and message: Merged build finished.
Test FAILed.

**Comment by cazino** [comment]
Retest this please.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/swift-tox/180/
Test PASSed.

**Comment by NicolasT** [comment]
You made this PR against `master` :wink:

**Comment by cazino** [comment]
Closing that one and create a new against 0.3.

**Pull request closed without merging**